esp/src/src/index.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import deepOrange from "@material-ui/core/colors/deepOrange";
+import { Frame } from "./react/frame";
+import { globalKeyValStore } from "./KeyValStore";
+import { initSession } from "./Session";
+
+import "css!hpcc/css/ecl.css";
+import "css!dojo-themes/flat/flat.css";
+import "css!hpcc/css/hpcc.css";
+
+declare const dojoConfig: any;
+
+const baseHost = "";
+const hashNodes = location.hash.split("#");
+
+dojoConfig.urlInfo = {
+    baseHost,
+    pathname: location.pathname,
+    hash: hashNodes.length >= 2 ? hashNodes[1] : "",
+    resourcePath: baseHost + "/esp/files/eclwatch",
+    basePath: baseHost + "/esp/files",
+    fullPath: location.origin + "/esp/files"
+};
+
+const store = globalKeyValStore();
+store.set("", "", false, "HPCCApps", "ECLWatch");
+initSession();
+
+const theme = createMuiTheme({
+    palette: {
```

Comment:
What theme are you adding here? Is this a custom theme just for this component?

Reply:
No idea - its just cut and paste from the material-ui Getting Started App.  This PR is about the DojoComponent, not about the index.html/index.tsx as such.